static/js/tippyjs.js

```diff
@@ -34,9 +44,10 @@ tippy.setDefaultProps({
     // tooltips.
     appendTo: "parent",
 
-    // html content is not supported by default
-    // enable it by passing data-tippy-allowHTML="true"
-    // in the tag or a parameter.
+    // To add a text tooltip, override this by setting data-tippy-content.
+    // To add an HTML tooltip, set data-tooltip-id to the id of a <template>.
+    // Or, override this with a function returning string or DocumentFragment.
```

Comment:
Do we want to explicitly say "returning an HTML string" here?

Reply:
Without `allowHTML`, Tippy interprets a string as text, not HTML.